symbol sets/src/ISOM2017_15000.xmap

```diff
@@ -155,58 +155,72 @@
             <cmyk method="spotcolor"/>
             <rgb method="spotcolor" r="0.240" g="1.000" b="0.090"/>
         </color>
-        <color priority="21" name="Green 50%" c="0.380" m="0.000" y="0.455" k="0.000" opacity="1.000">
+        <color priority="21" name="Green 60%" c="0.456" m="0.000" y="0.546" k="0.000" opacity="1.000">
+            <spotcolors knockout="true">
+                <component factor="0.6" spotcolor="14"/>
+            </spotcolors>
+            <cmyk method="spotcolor"/>
+            <rgb method="cmyk" r="0.544" g="1.000" b="0.454"/>
+        </color>
+        <color priority="22" name="Green 50%" c="0.380" m="0.000" y="0.455" k="0.000" opacity="1.000">
             <spotcolors knockout="true">
                 <component factor="0.5" spotcolor="14"/>
             </spotcolors>
             <cmyk method="spotcolor"/>
             <rgb method="spotcolor" r="0.620" g="1.000" b="0.545"/>
         </color>
-        <color priority="22" name="Green 20%" c="0.152" m="0.000" y="0.182" k="0.000" opacity="1.000">
+        <color priority="23" name="Green 30%" c="0.228" m="0.000" y="0.273" k="0.000" opacity="1.000">
+            <spotcolors knockout="true">
+                <component factor="0.3" spotcolor="14"/>
+            </spotcolors>
+            <cmyk method="spotcolor"/>
+            <rgb method="cmyk" r="0.772" g="1.000" b="0.727"/>
+        </color>
+        <color priority="24" name="Green 20%" c="0.152" m="0.000" y="0.182" k="0.000" opacity="1.000">
```

Comment:
@krticka  I guess that *all* uses of "Green 50%" and "Green 20%" (including forest runnable in one direction, narrow ride) are to be replaced by "Green 60%" and "Green 30%", respectively, not only 406, 408?

BTW I would appreciate if the MC could clarify the spelling of its specs (ISOM2017 vs. ISOM 2017 vs. ISOM-2017).

Reply:
The new changes in ISOM will be published officially as ISOM 2019.

Reply:
> The new changes in ISOM will be published officially as ISOM 2019.

When?

And what we should with ISOM 2017 symbol set now? Left it as is (but rename `ISOM2017_15000.xmap` to `ISOM_2017_15000.xmap`) and move all changes into separate `ISOM_2019_15000.xmap`?

Reply:
Well, for now ISOM_2017_15000 is OK. When ISOM 2019 will be published officially just rename the file. Maybe there will be some additional small corrections from MC meeting in January based upon the requests by federations/commissions. At least you have to wait with ISOM 2019 for official PDF where dimensions of new symbols will be defined.

Reply:
> just rename the file.

No, for *OpenOrienteering Mapper* we should create new file `ISOM_2017_15000.xmap` for each new specification & keep all legacy symbol sets (such as `ISOM_2000_15000.xmap`, `ISOM_2017_15000.xmap`, etc.) for compatibility reasons.

> At least you have to wait with ISOM 2019 for official PDF where dimensions of new symbols will be defined.

I can't understand why MC published changes as [ISOM-2017-corrections-approved-2018-11.pdf](https://orienteering.org/wp-content/uploads/2018/11/ISOM-2017-corrections-approved-2018-11.pdf), and not as `ISOM_2019_draft_2018-11.pdf`?!

Really misunderstanding  :-/